tests/finance/test_slippage.py

```diff
@@ -105,6 +106,28 @@ def test_equality_and_comparison(self):
         self.assertEqual(vol1.__dict__, vol1.asdict())
         self.assertEqual(vol2.__dict__, vol2.asdict())
 
+    def test_allowed_asset_types(self):
+        # Custom equities model.
+        class MyEquitiesModel(EquitySlippageModel):
+            def process_order(self, data, order):
+                return 0, 0
+
+        self.assertEqual(MyEquitiesModel.allowed_asset_types, (Equity,))
+
+        # Custom futures model.
+        class MyFuturesModel(FutureSlippageModel):
+            def process_order(self, data, order):
+                return 0, 0
+
+        self.assertEqual(MyFuturesModel.allowed_asset_types, (Future,))
+
+        # Custom model for both equities and futures.
+        class MyMixedModel(EquitySlippageModel, FutureSlippageModel):
+            def process_order(self, data, order):
+                return 0, 0
+
+        self.assertEqual(MyMixedModel.allowed_asset_types, (Equity, Future))
```

Comment:
The are great. Should we document (i.e. test) what happens if your custom model subclasses just `SilppageModel`, or how a custom model that defines `allowed_asset_types` itself interacts with its base classes?